src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -431,11 +433,28 @@ private function addWorkflowSection(ArrayNodeDefinition $rootNode): void
                                         ->prototype('scalar')
                                             ->cannotBeEmpty()
                                             ->validate()
-                                                ->ifTrue(fn ($v) => !class_exists($v) && !interface_exists($v, false))
+                                                ->ifTrue(static fn ($v) => !class_exists($v) && !interface_exists($v, false))
                                                 ->thenInvalid('The supported class or interface "%s" does not exist.')
                                             ->end()
                                         ->end()
                                     ->end()
+                                    ->arrayNode('definition_validators')
+                                        ->prototype('scalar')
+                                            ->cannotBeEmpty()
+                                            ->validate()
+                                                ->ifTrue(static fn ($v) => !class_exists($v))
+                                                ->thenInvalid('The validation class %s does not exist.')
+                                            ->end()
+                                            ->validate()
+                                                ->ifTrue(static fn ($v) => !is_a($v, DefinitionValidatorInterface::class, true))
+                                                ->thenInvalid(\sprintf('The validation class %%s is not an instance of "%s".', DefinitionValidatorInterface::class))
+                                            ->end()
+                                            ->validate()
+                                                ->ifTrue(static fn ($v) => 1 <= (new \ReflectionClass($v))->getConstructor()?->getNumberOfRequiredParameters())
+                                                ->thenInvalid('The validation class %s constructor must not have any arguments.')
```

Comment:
```suggestion
                                                ->thenInvalid('The "%s" validation class constructor must not have any arguments.')
```

Reply:
I updated this to 
```php
->thenInvalid('The %s validation class constructor must not have any arguments.')
```

without double quote, because they are automatically added by the config component.